content/posts/2020-06-14-how-does-the-pipe-operator-actually-work.Rmarkdown

```diff
@@ -67,7 +67,7 @@ identity(y)
 identity(mean(1:10))
 ```
 
-Using `substitute()` you can infer the actual *expression* that was passed to an argument. Put differently, you bypass evaluating the expression but instead *quote* it.
+Using `substitute()` you can capture the actual *expression* that was passed to an argument. Put differently, you bypass evaluating the expression but instead *quote* it.
```

Comment:
_infer_ may imply logical deduction inside the programmer's mind. _capture_ a more common term.